.travis.yml

```diff
@@ -3,16 +3,16 @@ install:
   - pip install tox
   - python3 setup.py egg_info
   - pip install -r aiosmtpd.egg-info/requires.txt
-matrix:
+jobs:
   include:
     - python: "3.6"
       env: INTERP=py36
     - python: "3.7"
       env: INTERP=py37
     - python: "3.8"
       env: INTERP=py38
-    - python: "pypy3"
-      env: INTERP=pypy3
```

Comment:
Taking this out temporarily because the current `pypy3` version on Travis CI is `7.1.1-beta0` and it had _not_ implemented bpo-27931 fix.